src/main/java/org/folio/util/UrlUtil.java

```diff
@@ -17,6 +20,22 @@ private UrlUtil() {
 
   }
 
+  /**
+   * Return the path component of the okapiUrl without tailing '/',
+   * return empty string on parse error.
+   */
+  public static String getPathFromOkapiUrl(String okapiUrl) {
+    try {
+      var path = new URL(okapiUrl).getPath();
+      if (path.endsWith("/")) {
+        return path.substring(0, path.length() - 1);
+      }
+      return path;
+    } catch (MalformedURLException e) {
+      return "";
+    }
+  }
```

Comment:
Please see my comment on mod-login here: https://github.com/folio-org/mod-login/pull/163#discussion_r1765070529. Have the same concern here.